shared/utils/generate-from-templates.py

```diff
@@ -51,59 +52,59 @@ def build(self):
                 script = self._get_script_for_csv(csv_filename)
 
                 csv_filepath = os.path.join(self._get_csv_dir(), csv_filename)
-                sys.stderr.write("{0}\t{1}\n".format(os.path.realpath(script), csv_filepath))
+                sys.stderr.write(
+                    "{0}\t{1}\n".format(os.path.realpath(script), csv_filepath)
+                )
                 self._run_script(script, csv_filepath)
 
     def input(self):
-        for file in self.get_input_list():
-            print(file)
+        for file_ in self.get_input_list():
+            print(file_)
 
     def output(self):
-        for file in self.get_output_list():
-            print(file)
-
-    def clean(self):
-        dir = self._parent_output_dir()
-        if os.path.exists(dir):
-            sys.stderr.write("Remove: {}\n".format(dir))
-            shutil.rmtree(dir)
+        for file_ in self.get_output_list():
+            print(file_)
 
     def get_input_list(self):
-        list = []
+        list_ = []
 
         for oval in self.supported_ovals:
             self._set_current_oval(oval)
 
             csv_dir = self._get_csv_dir()
             for csv in self._get_csv_list():
                 csv_filepath = os.path.join(csv_dir, csv)
-                script_filepath =  self._get_script_for_csv(csv)
+                script_filepath = self._get_script_for_csv(csv)
 
                 list.append(csv_filepath)
                 list.append(script_filepath)
 
                 for lang in self.langs:
-                    files_list = self._read_io_files_list(script_filepath, csv_filepath, lang, True)
-                    list.extend(files_list)
+                    files_list = self._read_io_files_list(
+                        script_filepath, csv_filepath, lang, True
+                    )
+                    list_.extend(files_list)
 
-        return self._deduplicate(list)
+        return self._deduplicate(list_)
```

Comment:
I'm not a python expert, but it seems odd that the list.append() and list.extend() lines aren't changed to "list_", but the array initialization and return lines do have this change.  If this is a throwaway variable, then can't we simply empty out the function ?  In any case, if this is expected to be a permanent change, and if the answer is not obvious to people with more python experience than me, then I wonder whether a descriptive inline comment might be appropriate.